Sources/BraintreePaymentFlow/BTLocalPaymentRequest.swift

```diff
@@ -167,7 +167,7 @@ extension BTLocalPaymentRequest: BTPaymentFlowRequestDelegate {
             apiClient.post("v1/local_payments/create", parameters: params) { body, response, error in
                 if let error {
                     if (error as NSError).code == BTCoreConstants.networkConnectionLostCode {
-                        apiClient.sendAnalyticsEvent("ios.local-payment-methods.network-connection.failure")
+                        apiClient.sendAnalyticsEvent("local-payment:\(BTPaymentFlowAnalytics.paymentNetworkConnectionLost)")
```

Comment:
How come this isn't using the helper method defined on line 265?

Reply:
I just wanted to clarify that this is using apiClient parameter in the `handle` function as opposed to the calls in `handleOpen` function where apiClient is not passed in as a parameter, it is referenced as `paymentFlowClientDelegate?.apiClient()`. They are the same but I wanted to keep the same logic.